src/gobapi/storage.py

```diff
@@ -63,7 +63,8 @@ def _get_table_and_model(collection_name, view):
     if view:
         return Table(view, metadata, autoload=True), None
     else:
-        return getattr(Base.classes, collection_name), GOBModel().get_model(collection_name)
+        return getattr(Base.classes, f'{catalog_name}_{collection_name}'), \
+                       GOBModel().get_collection(catalog_name, collection_name)
```

Comment:
I propose to defined a get_table_name(catalog_name, collection_name) method in gob_core so that the knowledge about the table name is kept at one place